.gitignore

```diff
@@ -2,6 +2,8 @@
 .eslintcache
 *.swp
 *~
+.idea
+*.iml
```

Comment:
do you mind removing these here and instead moving these into a global gitignore config for your computer?